pandas/tests/computation/test_eval.py

```diff
@@ -2035,6 +2035,15 @@ def test_truediv_deprecated(engine, parser):
     assert match in str(m[0].message)
 
 
+@pytest.mark.parametrize("column", ["Temp(°C)", "Capacitance(μF)"])
+def test_query_token(engine, column):
```

Comment:
Can you please add a comment referencing this pr here?

Reply:
and done